srcpkgs/ksh/template

```diff
@@ -4,14 +4,15 @@ version=2020.0.0
 revision=1
 wrksrc="ast-${version}"
 build_style=meson
-configure_args="-Dbuild-api-tests=false -Dbuild-api-tests-only=true"
+build_helper="qemu"
+configure_args="-Dbuild-api-tests=false -Dbuild-api-tests-only=true
+ -Dexe_wrapper=qemu-${XBPS_TARGET_QEMU_MACHINE}-static"
```

Comment:
I think this is added by default by the build style, isn't it? In the toolchain file.